src/sql/base/parts/tree/browser/treeView.ts

```diff
@@ -392,7 +391,6 @@ function reactionEquals(one: _.IDragOverReaction, other: _.IDragOverReaction | n
 export class TreeView extends HeightMap {
 
 	static readonly BINDING = 'monaco-tree-row';
-	static readonly LOADING_DECORATION_DELAY = 800;
```

Comment:
Why did you remove this?

Reply:
It was causing the refresh icon to not appear as often when there was this timeout. I removed the timeout so that the refresh would be more responsive to when the refresh action was taking place. If it is okay to only have the refresh icon appear occasionally (and to avoid brief flashes), I can restore it back to how it was before.

Reply:
I'm wondering if we should just reduce the delay instead. 800ms does seem like quite a long time to wait.

But I'm fine removing it completely for now - having it show even for a brief flash is pretty normal behavior for refresh actions in my experience.

Reply:
I'll restore the timeout delay to about 100ms as a middle ground.